### PR TITLE
Pd digital link remove

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?
Remove Digital Subscription from Nav

When V1 launches, we will no longer be selling the digital subscription. However, there are multiple references to it on the support site and these must be removed shortly before launch of the V1 checkout

Released behind the ab-newProduct 0% AB test so they're visible to the variant only. This AB test has already been set-up and is in use for showing changes to the checkout, you can force the AB test to be visible as so: https://support.theguardian.com/uk/contribute#ab-newProduct=variant

[**Trello Card**](https://trello.com/c/qXSZM0bu/568-support-site-remove-digital-subscription-from-nav)

## Why are you doing this?
image

Screenshot 2022-08-26 at 10 52 06

Screenshot 2022-08-26 at 10 52 36

Screenshot 2022-08-26 at 10 52 53

Screenshot 2022-08-26 at 16 01 17

Screenshot 2022-08-26 at 16 01 51

Screenshot 2022-08-26 at 16 02 15

## Is this an AB test?
- [x] Yes
- [ ] No

If this is an AB test, PR reviewers should open and check the Optimize test.
[**Optimize Link**](https://optimize.google.com/optimize/home)

### CODE REVIEW

**Comment by GHaberis** [comment]
@paul-daniel-dempsey does this also hide the link from the mobile/tablet version of the navigation? No worries if not (it wasn't mentioned in the original card!). If not I'll create another card to do that in separate PR.

Screenshot 2022-08-26 at 14 51 39

**Comment by paul-daniel-dempsey** [comment]
Oh no! Didn't think to make that change in my rush to get it in before standup. Shall I move it back and just look at that in this existing card?

**Comment by GHaberis** [comment]
> Oh no! Didn't think to make that change in my rush to get it in before standup. Shall I move it back and just look at that in this existing card?

Have a quick look at the code, if you think it's straight forward and not a lot of extra code then put it in this PR, otherwise we can do it in a separate PR.

**Comment by paul-daniel-dempsey** [comment]
> > Oh no! Didn't think to make that change in my rush to get it in before standup. Shall I move it back and just look at that in this existing card?
> 
> Have a quick look at the code, if you think it's straight forward and not a lot of extra code then put it in this PR, otherwise we can do it in a separate PR.

Will have a look now and pull card back

**Comment by paul-daniel-dempsey** [comment]
Ok, small change done, now to figure out what to do with this request (or pull an new one)

**Comment by GHaberis** [comment]
> Ok, small change done, now to figure out what to do with this request (or pull an new one)

We can still use this pull request, it's ok to add changes after the PRs been raised, so no need to do anything other than wait for a review!

File: support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ function SubscriptionsLandingPage({
 	pricingCopy,
 	referrerAcquisitions,
 }: SubscriptionsLandingPropTypes) {
+	const hideDigital = participations.newProduct === 'variant' ? true : false;
```
Comment (i-hardy) [inline support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.tsx:28]:
```suggestion
	const hideDigital = participations.newProduct === 'variant';
```

File: support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.tsx [hunk 2]
```diff
@@ -67,6 +67,7 @@ function WeeklyLandingPage({
 	const sanitisedPromoCopy = getPromotionCopy(promotionCopy);
 	// ID for Selenium tests
 	const pageQaId = `qa-guardian-weekly${orderIsAGift ? '-gift' : ''}`;
+	const hideDigital = participations.newProduct === 'variant' ? true : false;
```
Comment (i-hardy) [inline support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.tsx:70]:
```suggestion
	const hideDigital = participations.newProduct === 'variant';
```

File: support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx [hunk 3]
```diff
@@ -45,7 +45,9 @@ const pageQaId = 'qa-paper-subscriptions';
 function PaperLandingPage({
 	productPrices,
 	promotionCopy,
+	participations,
 }: PaperLandingPropTypes) {
+	const hideDigital = participations.newProduct === 'variant' ? true : false;
```
Comment (i-hardy) [inline support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx:50]:
```suggestion
	const hideDigital = participations.newProduct === 'variant';
```

**Review by i-hardy — APPROVED** [review]
This looks great, just a small suggestion on checking for the variant

File: support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx (hunk 3 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx:50]: Of course, will neaten a bit further now, thanks for the review!

Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx]: All done, if you get a minute, etc etc, ta

**Pull request merged**